sub_composeEff_rooKeys.sub

```diff
@@ -13,6 +13,6 @@ totdiv      = 50
 Arguments   = $INT(bin) $INT(indx) $INT(par) $(wid0) $(wid1) $(wid2) $INT(xbin) $INT(ybin) $INT(zbin) $INT(ndiv) $INT(totdiv)
 Log         = logs_parSub/sub_$(ClusterId).log
 Output      = logs_parSub/composeEff_rooKeys_$INT(bin)_$INT(indx)_$INT(par)_$(wid0)_$(wid1)_$(wid2)_$INT(xbin)_$INT(ybin)_$INT(zbin)_$INT(ndiv)_$INT(totdiv).out
-Error       = logs_parSub/composeEff_rooKeys_$INT(bin)_$INT(indx)_$INT(par)_$(wid0)_$(wid1)_$(wid2)_$INT(xbin)_$INT(ybin)_$INT(zbin)_$INT(ndiv)_$INT(totdiv).out
+Error       = logs_parSub/composeEff_rooKeys_$INT(bin)_$INT(indx)_$INT(par)_$(wid0)_$(wid1)_$(wid2)_$INT(xbin)_$INT(ybin)_$INT(zbin)_$INT(ndiv)_$INT(totdiv).err
```

Comment:
Either we keep this single-submission script working (adding the year variable), or we remove it

Reply:
I would be in favour of removing it, but please comment.

Reply:
Its only utility is when not all the parallel jobs give a valid result, and a partial result is merged. In this case with this macro it is easy to resubmit only one or few jobs

Reply:
done